UnitsNet.Tests/CustomCode/EnergyTests.cs

```diff
@@ -30,5 +30,55 @@ public class EnergyTests : EnergyTestsBase
         protected override double JoulesInOneJoule { get { return 1; } }
         protected override double KilojoulesInOneJoule { get { return 1E-3; } }
         protected override double MegajoulesInOneJoule { get { return 1E-6; } }
+
+        protected override double BritishThermalUnitsInOneJoule
+        {
+            get { return 1/1055.05585262; }
```

Comment:
I think you're kind of cheating here, since you basically just copied in the implementation into the test. The idea is to double check that you got it right, by looking up the correct answer for these particular conversions.

For example, I type this in google:
`british termal units in 1 joule` I get `1 joule = 0.00094781712 british thermal units`, so I would put that number in here.